copyparty/web/browser.js

```diff
@@ -8185,6 +8185,635 @@ var Ls = {
 
 		"lang_set": "¿refrescar para que el cambio surta efecto?"
 	},
+	"swe": {
+		"tt": "Svenska",
+
+		"cols": {
+			"c": "aktion",
+			"dur": "längd",
+			"q": "kvalitet / bitrate",
+			"Ac": "ljudkodek",
+			"Vc": "videokodek",
+			"Fmt": "format / container",
+			"Ahash": "ljudchecksumma",
+			"Vhash": "videochecksumma",
+			"Res": "upplösning",
+			"T": "filtyp",
+			"aq": "ljudkvalitet / bitrate",
+			"vq": "videokvalitet / bitrate",
+			"pixfmt": "subsampling / pixelstruktur",
+			"resw": "horisontell upplösning",
+			"resh": "vertikal upplösning",
+			"chs": "antal ljudkanaler",
+			"hz": "samplingsfrekvens"
+		},
+
+		"hks": [
+			[
+				"övrigt",
+				["ESC", "stäng diverse paneler"],
+
+				"filhanterare",
+				["G", "växla mellan listvy / rutnät"],
+				["T", "växla mellan miniatyrer / ikoner"],
+				["⇧ A/D", "miniatyrstorlek"],
+				["ctrl-K", "radera urval"],
+				["ctrl-X", "klipp urval till urklipp"],
+				["ctrl-C", "kopiera urval till urklipp"],
+				["ctrl-V", "klistra in (kopiera/flytta) hit"],
+				["Y", "ladda ner urval"],
+				["F2", "byt namn på urval"],
+
+				"välja filer",
+				["Blanksteg", "växla val av fil"],
+				["↑/↓", "flytta filvalsmarkör"],
+				["ctrl ↑/↓", "flytta markör och vy"],
+				["⇧ ↑/↓", "välj föregående/nästa fil"],
+				["ctrl-A", "välj alla filer / mappar"],
+			], [
+				"navigation",
+				["B", "växla mellan brödsmulor / trädvy"],
+				["I/K", "föregående/nästa mapp"],
+				["M", "hoppa till överordnad mapp (eller kollapsa närvarande)"],
```

Comment:
"Överordnad mapp" sounds incredibly formal to me, but it's apparently what it's called.

Reply:
could probably change the phrasing to something like "jump to the directory above", but in swedish?

Reply:
I think we'll just keep this for now unless someone really objects to it.